SpatialGDK/Source/SpatialGDK/Private/Interop/SkeletonEntityCreationStep.h

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "Interop/SkeletonEntities.h"
+
+class USpatialNetDriver;
+
+namespace SpatialGDK
+{
+class FSkeletonEntityCreationStep
```

Comment:
I'd call this something other than step and have a comment describing what it is.